addon/components/x-select.js

```diff
@@ -62,11 +62,11 @@ export default Ember.Component.extend({
    *   {{!options here ....}}
    * {{/x-select}}
    *
-   * @property oneWay
+   * @property one-way
    * @type Boolean
    * @default false
    */
-  oneWay: false,
+  'one-way': false,
```

Comment:
Can we create a line below this that aliases `oneWay` to `one-way` but only ref `one-way` in the docs? Only because I cut the release of 2.2 already and this would break semver :( 

```js
"one-way": Ember.computed.alias('oneWay'),
```